drivers/vulkan/vulkan_context.cpp

```diff
@@ -1892,7 +1892,7 @@ Error VulkanContext::_update_swap_chain(Window *window) {
 		window->vsync_mode = DisplayServer::VSYNC_ENABLED; // Set to default.
 	}
 
-	print_verbose("Using present mode: " + String(string_VkPresentModeKHR(window->presentMode)));
+	WARN_PRINT_ONCE("Using present mode: " + String(string_VkPresentModeKHR(window->presentMode)));
```

Comment:
That's an information message AFAIK, it shouldn't be a warning. This would also make it always visible instead of verbose only.

If it's not useful, it should just be removed.

Reply:
Only reason why I made it as a warning that prints once, is because there were no macros for simple prints to fire once. Maybe removing this message altogether would be reasonable, not sure why would the end user need it besides debugging some Vulkan issue.

Reply:
I'd suggest keeping the `print_verbose()`, but only do it if the present mode is different from the last one.